### PR TITLE
Add present job.  FHIR-46274

### PR DESCRIPTION
**Pull request by saulakravitz**
Replace the UsualWork with new PresentJob
[FHIR-46274](https://jira.hl7.org/browse/FHIR-46274)

### CODE REVIEW

File: input/fsh/extensions/ExtensionRole.fsh
Comment (ptran265) [inline input/fsh/extensions/ExtensionRole.fsh:1]: Is this a necessary addition? Seems the profile uses ExtensionRoleVitalRecords and this ExtensionRole doesn't get referenced

Comment (saulakravitz) [reply input/fsh/extensions/ExtensionRole.fsh:1]: ExtensionRoleVitalRecords isn't needed.   Should go away.  Only place it was used was in the UsualWorkVitalRecords.

**Review by rpassas — COMMENTED** [review]
Changes were made in the master sheet repo - please re-run script to make sure it aligns with what you change here

**Review by rpassas — COMMENTED** [review]
Which form does the usual work information come from in the natality / fetal death cases? I am not seeing it

**Comment by saulakravitz** [comment]
UsualWork --> PresentJob

**Comment by saulakravitz** [comment]
Re Master csv vs local csv....
I thought I copied the master sheet into this repo....

**Review by rpassas — APPROVED** [review]

**Pull request merged**